### PR TITLE
LLSignalTestRecorder: add thread verification support.

### PR DESCRIPTION
**Pull request by barakyoresh**

### CODE REVIEW

File: LLReactiveMatchers/Helpers/LLSignalTestRecorder.m [hunk 1]
```diff
@@ -26,6 +26,7 @@ @interface LLSignalTestRecorder ()
 @property (nonatomic, strong) NSMutableArray *receivedEvents;
 @property (nonatomic, strong) NSError *receivedError;
 
+@property (nonatomic, strong) NSMutableSet *activeThreadsInReceivedEvents;
```
Comment (StatusReport) [inline LLReactiveMatchers/Helpers/LLSignalTestRecorder.m:29]: Newline.

File: LLReactiveMatchers/Helpers/LLSignalTestRecorder.m [hunk 2]
```diff
@@ -162,6 +167,16 @@ - (BOOL) haveFinished {
...
 }
 
+- (NSSet *)operatingThreads {
+  @synchronized(self) {
+    return [self.activeThreadsInReceivedEvents copy];
+  }
+}
+
+- (NSUInteger) operatingThreadsCount {
+  return self.activeThreadsInReceivedEvents.count;
```
Comment (StatusReport) [inline LLReactiveMatchers/Helpers/LLSignalTestRecorder.m]: Why no `@synchronized` here?

**Comment by StatusReport** [comment]
Fix and squash

File: LLReactiveMatchers/Helpers/LLSignalTestRecorder.m (hunk 1 above)
Comment (barakyoresh) [inline LLReactiveMatchers/Helpers/LLSignalTestRecorder.m:29]: Done.

File: LLReactiveMatchers/Helpers/LLSignalTestRecorder.m (hunk 2 above)
Comment (barakyoresh) [inline LLReactiveMatchers/Helpers/LLSignalTestRecorder.m]: Type, needs to be `self.operatingThreads`. Fixed.

**Comment by barakyoresh** [comment]
Fixed and squashed.

**Pull request merged**